app/models/dhs1171_pdf.rb

```diff
@@ -0,0 +1,38 @@
+class Dhs1171Pdf
+  FIELDS = [
+    "applying_for_food_assistance",
+    "birth_day",
+    "birth_month",
+    "birth_year",
+    "city",
+    "county",
+    "full_name",
+    "signature",
+    "signature_date",
+    "state",
+    "street_address",
+    "zip",
+  ].freeze
+
+  def initialize(client_info)
+    @source_pdf = "DHS_1171.pdf"
+    @client_info = client_info
+  end
+
+  def save(new_pdf)
```

Comment:
Just to clarify, `new_pdf` here is the new pdf file name? Maybe `new_pdf_filename` if that's the case? Or `input_pdf_filename` and `output_pdf_filename`? or `source` vs `destination`?